Anime-App/Feature/Anime/ViewModel/AnimeViewModel.swift

```diff
@@ -8,39 +8,74 @@
 import Foundation
 
 class AnimeViewModel {
-        
+    
     private(set) var animes: [Anime] = []
+    private var queryObject = QueryObject()
+
  
+    func fetchAnime(page: Int, completion: @escaping (Result<Void,NetworkingError>) -> Void) {
+        
+        guard var urlComps = URLComponents(string:  BaseUrl.baseUrl.url + UrlEndpoint.getAnime.endpoint + UrlEndpoint.pagination.endpoint) else { return }
```

Comment:
This is still wrong can you follow this tutorial, all of the URL configuration should also be in in object and configure there. Then it should just return a URL https://www.swiftbysundell.com/articles/constructing-urls-in-swift/